### ISSUE TITLE
macOS instructions for Ubuntu Core assume command line utilities not available in macOS

### ISSUE THREAD

**Issue by huertanix** [canonical-web-and-design/ubuntu.com#5200]
On this page: https://developer.ubuntu.com/core/get-started/installation-medias, macOS instructions include a cryptic command `sudo sh -c 'xzcat ~/Downloads/<image file> | sudo dd of=<drive address> bs=32m'` which assumes macOS includes the `xzcat` utility. At least in macOS Sierra, it does not, and running this command causes 0 bytes to be written to the SD card and additionally requires unmounting it again before trying again--a recovery step not mentioned in the instructions. 

Attempting to use the built-in macOS Archive Utility (i.e. double-clicking it) to decompress the .xc file only results in the creation of a `ubuntu-core-16-pi3.img 2.xz.cpgz` file, and trying to decompress that only results back in a `ubuntu-core-16-pi3.img 2 2.xz` completing a full cycle of futility that is more a  hazing ritual than helpful plan B.

tl;dr please get rid of the use of xc, use zip and provide accurate macOS instructions.

**Comment by matthewpaulthomas** [comment]
I’m pretty much the worst person to assign this to, since I don’t have macOS at work so I can’t test any changes. What I can do is temporarily remove the macOS instructions, on the grounds that broken instructions are worse than none at all.

**Comment by pmahnke** [comment]
@matthewpaulthomas it seems that gunzip should work for most modern macs

`sudo sh -c 'gunzip ~/Downloads/<image file> | sudo dd of=<drive address> bs=32m'`

we could test on @MeltySnow's mac

**Comment by matthewpaulthomas** [comment]
Cunningham’s Law strikes again!

**Comment by matthewpaulthomas** [comment]
The revised instructions still need testing.

**Comment by pmahnke** [comment]
we tested on one mac, and this worked, so closing.

please reopen if someone finds it does not on another version of macOS
